### DISCUSSION TITLE
Add "include image" option into Notification settings. ngalert

### DISCUSSION THREAD

**Discussion by nklym** [grafana/grafana#42530] [Alerting]
**Why is this component needed**:
ngalert is a good continuation of the alerting in grafana, but getting alerts without images is inconvenient.
In my opinion "include image" is a very useful function.
___
 - [x] Is/could it be used in more than one place in Grafana?
_This feature is already available for simple alerts (without ngalert)_

**Where is/could it be used?**:
The **include image** should be shown in the "contact point" -> "Notification settings" configuration. 
/alerting/notifications/receivers/[NAME]/edit?alertmanager=[NAME]
![129883661-a7d34bd3-09f7-4ebe-a74c-3c9ee739799f](https://user-images.githubusercontent.com/58990524/132179436-8ad0beaa-9616-4e54-b22c-84801c7faeb0.png)
___
- [ ] Post screenshots possible.
- [ ] It has a single use case.
- [x] It is/could be used in multiple places.

**Implementation** (Checklist meant for the person implementing the component)

- [ ] Component has a story in Storybook.
- [x] Props and naming follows [our style guide](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md).
- [x] It is extendable (rest props are spread, styles with className work, and so on).
- [x] Uses [theme for spacing, colors, and so on](https://github.com/grafana/grafana/blob/main/contribute/style-guides/themes.md).
- [x] Works with both light and dark theme.

**Documentation**

- [ ] Properties are documented.
- [ ] Use cases are described.
- [ ] Code examples for the different use cases.
- [ ] Dos and don'ts.
- [ ] Styling guidelines, specific color usage (if applicable).

**Comment by gotjosh** [comment]
I'm turning this into a discussion as it is a feature request. **For context, we've completely deprecated the support for images in Alerting.** 

With enough support, I'm happy to start digging up more context on why we removed it and what kind of solutions we can propose moving forward.

But I don't think this is needed anymore. We've included a way to customise your notifications via templating and you get a direct link to the queries (including graphs) that generated this alert. With those two, the alert itself should have enough context to help you dig what you need.

**Comment by alsotop** [comment]
There appears to be three other discussion threads started about the same topic:
* https://github.com/grafana/grafana/discussions/38030
* https://github.com/grafana/grafana/discussions/41015
* https://github.com/grafana/grafana/discussions/38003

I would say that it makes sense to try and bring these topics together if we want to see that there is a growing demand for this feature to return.

I'd like to put out there that I agree with other posters that images in alerts (even if they are optional) can be a very powerful thing and something that I find has been stopping me from moving from legacy alerts to ngalert. The images provided a great way to really quickly get a lot of context about an issue that may be occurring, including some previous history of how the panel got to that point. I can appreciate that "Grafana 8 alerts are no longer directly tied to panel queries." ([ref](https://github.com/grafana/grafana/discussions/41015#discussioncomment-1548846)) but for where they are or can be, the image can be very useful, especially in situations such as Slack notifications.

![image](https://user-images.githubusercontent.com/9293693/144817675-a1266dce-0b0b-4daf-9c73-8b3517406eaf.png)

I'd like to discover how likely it is that Image Rendering will return to ngalerts. Given that Grafana 8.3.0 has now defaulted to ngalerts, I'd rather not stay stuck on the legacy alert system if image notifications are not anywhere near in the pipeline/future of Grafana development.

**Comment by gotjosh** [comment]
@alsotop thanks for digging up. Given https://github.com/grafana/grafana/discussions/38030 appears to be the done with the most upvotes. I'm going to close all the others in favour of this one to keep the discussion centralised.
